src/wallet/wallet_rpc_server.cpp

```diff
@@ -1889,8 +1889,15 @@ namespace tools
       return false;
     }
 
+    if (req.account_index >= m_wallet->get_num_subaddress_accounts())
+    {
+      er.code = WALLET_RPC_ERROR_CODE_ACCOUNT_INDEX_OUTOFBOUND;
```

Comment:
A bit picky, but... why remove the last two underscores ? And the usual phrasing is out of bounds (plural), even though you have just one bound here.

Reply:
Fixed :)